client/components/select-dropdown/separator.jsx

```diff
@@ -3,15 +3,10 @@
 /**
  * External Dependencies
  */
-var React = require( 'react' );
+import React from 'react';
 
-var SelectDropdownSeparator = React.createClass( {
-
-	render: function() {
-		return (
-			<li className="select-dropdown__separator" />
-		);
-	}
-} );
-
-module.exports = SelectDropdownSeparator;
+export default function SelectDropdownSeparator() {
```

Comment:
Yes, I'm sorry. you're right. What I meant was to remove the `return` if we use an arrow function:

```es6
const SelectDropdownSeparator = () => 
  <li className="select-dropdown__separator" />;

export default SelectDropdownSeparator;
```

Feel free to change this.